ui/src/app/components/environment-cost-estimator.tsx

```diff
@@ -42,16 +43,33 @@ const styles = reactStyles({
   },
 });
 
+// To simplify calculations and the UI, we assume the workspace has only one GKE app running.
+const GKE_APP_HOURLY_USD_COST_PER_WORKSPACE = 0.2;
+
 export const EnvironmentCostEstimator = ({
   analysisConfig,
+  isGKEApp,
   costTextColor = colors.accent,
   style = {},
 }: Props) => {
   const { detachedDisk, diskConfig } = analysisConfig;
-  const runningCost = machineRunningCost(analysisConfig);
+  const runningCost =
+    machineRunningCost(analysisConfig) +
+    (isGKEApp ? GKE_APP_HOURLY_USD_COST_PER_WORKSPACE : 0);
   const runningCostBreakdown = machineRunningCostBreakdown(analysisConfig);
-  const storageCost = machineStorageCost(analysisConfig);
-  const storageCostBreakdown = machineStorageCostBreakdown(analysisConfig);
+  const pausedCost =
```

Comment:
I was confused why non-GKE app(GCE and dataproc) paused cost is 0, then I realize that the actual GCE cost might be covered inside `pausedCostBreakdown`?

Reply:
Here's the screenshots for Jupyter. It's the same before and after this change.

General Analysis:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/31020403/230994552-d386dae6-bbf1-48fd-b6d8-956f093e685b.png">

Hail Genomics Analysis:
<img width="684" alt="image" src="https://user-images.githubusercontent.com/31020403/230994599-58e38a6b-c194-4004-800e-ca7ad16942cf.png">